### PR TITLE
Log support

### PR DESCRIPTION
**Pull request by superjax**
slight modification to extra sensor support, added log message support back to main body of code.

### CODE REVIEW

File: src/mavlink.cpp [hunk 1]
```diff
@@ -416,37 +427,37 @@ void Mavlink::send_attitude(void)
...
+                             gyro.y,
+                             gyro.z,
+                             RF_.sensors_.data().imu_temperature);
+  send_message(msg);
+  //  }
+  //  else
+  //  {
+  // Otherwise, wait and signal that we still need to send IMU
+  //    mavlink_streams[STREAM_ID_IMU].next_time_us -= mavlink_streams[STREAM_ID_IMU].period_us;
+  //  }
```
Comment (dpkoch) [inline src/mavlink.cpp]: Just double checking, is this stuff supposed to be commented out?

Comment (superjax) [inline src/mavlink.cpp]: Yeah, that is on purpose (for now).   That little routine is to make sure that we don't send the same IMU measurement more than once.

File: src/sensors.cpp [hunk 2]
```diff
@@ -59,80 +59,115 @@ void Sensors::init()
...
+      float mag[3];
+      rf_.board_.mag_read(mag);
+      data_.mag.x = mag[0];
+      data_.mag.y = mag[1];
+      data_.mag.z = mag[2];
+      correct_mag();
+    }
+    break;
   }
+  next_sensor_to_update_ = (next_sensor_to_update_ + 1) % 4;
```
Comment (superjax) [inline src/sensors.cpp]: Yeah.  I do too.  It just seemed like a lot of overhead at the time.  Perhaps it would make it cleaner

**Comment by superjax** [comment]
Yeah, the sensor streaming still works.

**Pull request merged**